tokens/tests/structureForStyleDictionary.spec.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const { structureForStyleDictionary } = require( './../build/wikimedia-ui-base/lib' );
+
+describe( 'structureForStyleDictionary', () => {
+	it( 'prefixes imported variables', function () {
```

Comment:
Let's make this arrow syntax so that we have a good example for copy-paste development (me).

Reply:
Done